src/pages/generate/pages/result/components/GeneratedImgA.tsx

```
@@ -0,0 +1,248 @@
+            <img
+              src={image.imageUrl}
+              alt={`${image.name}님을 위한 맞춤 인테리어 스타일링`}
+              className={styles.imgArea({ mirrored: image.isMirror })}
+            />
+          </SwiperSlide>
+        ))}
+        {shouldAddBlurredSlide && (
+          <SwiperSlide key="blurred-second-image">
+            <div
+              className={styles.imgAreaBlurred({
+                mirrored: images[1].isMirror,
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)